spring-boot-admin-server/src/main/java/de/codecentric/boot/admin/server/config/SpringBootAdminServerEnabledCondition.java

```diff
@@ -0,0 +1,50 @@
+/*
+ * Copyright 2014-2018 the original author or authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package de.codecentric.boot.admin.server.config;
+
+import org.springframework.boot.autoconfigure.condition.ConditionOutcome;
+import org.springframework.boot.autoconfigure.condition.SpringBootCondition;
+import org.springframework.boot.context.properties.bind.Bindable;
+import org.springframework.boot.context.properties.bind.Binder;
+import org.springframework.context.annotation.ConditionContext;
+import org.springframework.core.type.AnnotatedTypeMetadata;
+
+/**
+ * This condition checks if the sever should be enabled. Property
+ * spring.boot.admin.server.enabled is checked.
+ */
+public class SpringBootAdminServerEnabledCondition extends SpringBootCondition {
+
+	@Override
+	public ConditionOutcome getMatchOutcome(ConditionContext context, AnnotatedTypeMetadata annotatedTypeMetadata) {
+		AdminServerProperties serverProperties = getClientProperties(context);
+
+		if (!serverProperties.getServer().isEnabled()) {
+			return ConditionOutcome
+					.noMatch("Spring Boot Server is disabled, because 'spring.boot.admin.client.enabled' is false.");
```

Comment:
Shouldn't this be `spring.boot.admin.server.enabled` in the `noMatch` string?

Reply:
fixed in https://github.com/codecentric/spring-boot-admin/pull/1672/commits/de02259cfffb08ce802ef83473aeba9fbba1c4b6